.github/CODEOWNERS

```diff
@@ -3833,7 +3833,7 @@
 /types/kube-probe/                                      @emacsified
 /types/kue/                                             @drudge @amiram @pc-jedi @deerawan
 /types/kue-ui-client/                                   @bengry
-/types/kurento-client/                                  @jhukdev @michelalbers @joeflateau @whitphx
+/types/kurento-client/                                  @jahilldev @michelalbers @joeflateau @whitphx
```

Comment:
please revert this specific change, the file is autogenerated and that's not advised

Reply:
@peterblazejewicz Ooops! I've reverted this now 👍